microsetta_interface/implementation.py

```diff
@@ -995,15 +1007,37 @@ def get_update_sample(*, account_id=None, source_id=None, sample_id=None):
                         "Nares", "Nasal mucus", "Right hand", "Left hand",
                         "Forehead", "Torso", "Right leg", "Left leg",
                         "Vaginal mucus", "Tears", "Ear wax", "Hair", "Fur"]
+        # babel scraping doesn't understand anything but constant strings.
+        sample_site_translations = [
```

Comment:
Is that true though? Does the following not work?

```python
sample_site_translations = [gettext(s) for s in sample_sites]
```

...or is pybabel walking the AST to find `gettext` calls?

Reply:
It does not!  At least not when I tried it.  That was my first try.

Reply:
okay cool cool